ethers-etherscan/src/lib.rs

```diff
@@ -263,6 +264,13 @@ impl ClientBuilder {
             Chain::Moonriver => {
                 urls("https://api-moonriver.moonscan.io/api", "https://moonriver.moonscan.io")
             }
+            Chain::Emerald => {
+                urls("https://explorer.emerald.oasis.dev/api", "https://explorer.emerald.oasis.dev")
+            }
+            Chain::EmeraldTestnet => urls(
+                "https://testnet.explorer.emerald.oasis.dev/api",
+                "https://testnet.explorer.emerald.oasis.dev",
+            ),
```

Comment:
is this compatible with etherscan API?

Reply:
@mattsse This project is using the fork for a small POC: https://gunix.codeberg.page/sycamore-metamask-example/
However that's only getting the balance and nothing else.

The explorer is [blockscout](https://github.com/blockscout/blockscout). According to the docs, it [supports ETH RPC](https://docs.blockscout.com/for-users/api#eth-rpc)

Reply:
Yeah, we're using our own fork of the blockscout: https://github.com/oasisprotocol/emerald-blockscout

Let us know, if you require any assistance.